src/deriveAddress.c

```diff
@@ -165,6 +191,88 @@ static void deriveAddress_return_ui_runStep()
 	UI_STEP_END(RETURN_UI_STEP_INVALID);
 }
 
+/* ========================== DISPLAY ADDRESS ========================== */
+
+static void _displayVerifyAddressMsg(ui_callback_fn_t* this_fn)
+{
+	#ifdef HAVE_BAGL
+	ui_displayPaginatedText(
+	        "Verify address",
+	        "Make sure it agrees with your computer",
+	        this_fn
+	);
+	#elif defined(HAVE_NBGL)
+	set_light_confirmation(true);
+	display_warning(
+	        "Make sure address matches\nwith your computer",
+	        this_fn,
+	        respond_with_user_reject
+	);
+	#endif // HAVE_BAGL
+}
+
+static void _displaySpendingInfo_displayAddr(ui_callback_fn_t* this_fn)
+{
+	#ifdef HAVE_BAGL
+	ui_displaySpendingInfoScreen(&ctx->addressParams, this_fn);
+	#elif defined(HAVE_NBGL)
+#define SPENDING_INFO_SIZE MAX(11 + BECH32_PREFIX_LENGTH_MAX + 2 * BECH32_BUFFER_SIZE_MAX, 2 * BECH32_BUFFER_SIZE_MAX)
+	char line1[30] = {0};
+	char spendingInfo[SPENDING_INFO_SIZE] = {0};
+	ui_getSpendingInfoScreen(line1, SIZEOF(line1), spendingInfo, SIZEOF(spendingInfo), &ctx->addressParams);
+	fill_address_data(line1, spendingInfo);
```

Comment:
`this_fn` is not used here... Several other such places, getting a warning when compiling for Stax.